utils/build-script-impl

```diff
@@ -161,6 +161,7 @@ KNOWN_SETTINGS=(
     build-swift-examples        "1"              "set to 1 to build examples"
     build-serialized-stdlib-unittest "0"         "set to 1 to build the StdlibUnittest module with -sil-serialize-all"
     build-sil-debugging-stdlib  "0"              "set to 1 to build the Swift standard library with -gsil to enable debugging and profiling on SIL level"
+    build-sourcekit             ""               "set to build the SourceKit library and tools (enabled by default on Darwin hosts)"
```

Comment:
Could you lift the computation of the default value into the Python script?  Maybe as a separate patch.